setup.py

```diff
@@ -16,8 +16,8 @@
     keywords='Firebase',
     packages=find_packages(exclude=['tests']),
     install_requires=[
-        'requests', 'sseclient', 'gcloud',
-        'oauth2client', 'requests_toolbelt', 'python_jwt',
-        'pycrypto'
+        'requests==2.11.1', 'sseclient', 'gcloud==0.17.0',
```

Comment:
I see that you didn't add a version for sseclient. It probably shouldn't be there anymore, and I assume that when it was removed from requirements.txt it was just forgotten to remove it here too